### ISSUE TITLE
Open in external pdf viewer

### ISSUE THREAD

**Issue by johschmitz** [James-Yu/Atom-LaTeX#1]
There is no way to specify an external PDF viewer such as evince, okular and so on.

**Comment by James-Yu** [comment]
Atom-LaTeX does not support external PDF viewer and currently there is no future plan on that. That is the main reason why the feature is called preview.

Temporarily close the issue. Will review in the future if the feature is a must.

**Comment by p10n** [comment]
[OT: Thanks fpr this great package! However...:  ]
I understand why you include a pdf viewer in the package. However, especially on Mac OS X, the included viewer is far superior to what I am used to:

- Both the Apple-provided "Preview.app" as well as the great [Skim](https://sourceforge.net/projects/skim-app/) are much more responsive in zooming and panning. In both of them, zooming is perfectly smooth – while the Atom-LaTeX bult-in viewer is slower and feels shaky. (I have experienced cases where it got totally stuck when zooming into detail.)
- When the PDF gets re-compiled, the Atom-LaTeX viewer flashes to black for a while. This stresses the eye and makes it hard to compare the previous and newly compiled pdf for subtle differences (like image or text positioning).
- The Atom-LaTeX viewer seems to have problems with certain included images (in LaTeX beamer slides, some included pdf images are not displayed at all, and any text below the images on the slide appears to be missing). Might be caused by different PDF version of the included images. Anyway, no problem for both Skim and Apple's Preview.app.
- When compiling another LaTeX document, the PDF preview window of the previous one vanishes. (I admit this can well be a feature, but I would prefer letting me close it manually.)

To sum it up, Skim is by far superior, and it handles syncTeX / PDFSync nicely. To me personally, the Atom-LaTeX viewer feels like a re-invented wheel, and I very much long for an option to choose a custom PDF viewer. (I am fully ok with the Atom-LaTeX viewer being default, so that the package remains plug-and-play.)
